domain/src/test/java/org/oppia/android/domain/classify/rules/itemselectioninput/ItemSelectionInputDoesNotContainAtLeastOneOfRuleClassifierProviderTest.kt

```diff
@@ -10,14 +10,11 @@ import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.oppia.android.domain.classify.InteractionObjectTestBuilder
+import org.oppia.android.testing.assertThrows
 import org.robolectric.annotation.Config
 import org.robolectric.annotation.LooperMode
-import java.lang.IllegalStateException
```

Comment:
was this import an unused one before removing? 
checking if it is unused why ktlint didn't catch it!!

Reply:
I think Ktlint did not catch it because on adding this import, it is getting used wherever `IllegalStateException::class` is being used. But on optimizing the imports, it's getting removed since it isn't actually required, `IllegalStateException` is being taken from `TypeAliases.kt` even without the import.

Reply:
got it. Thanks